app/adapters/cedar-metadata-record.ts

```diff
@@ -0,0 +1,30 @@
+import Store from '@ember-data/store';
+import DS from 'ember-data';
+import RSVP from 'rsvp';
+
+import OsfAdapter from './osf-adapter';
+
+export default class CedarMetadataRecordAdapter extends OsfAdapter {
```

Comment:
We also need to add `namespace = '_'` here because we won't be using the default `v2` namespace for this endpoint.